### DISCUSSION TITLE
Access database connection string

### DISCUSSION THREAD

**Discussion by frankmyhre** [nextgenhealthcare/connect#6378] [Q&A]
Hi all, 
I am trying to connect to a local MS SQL database via Mirth. I have the username and password correct, but I just can't seem to connect to the database. My guess is that it is the connection string as below:

`jdbc:odbc:Driver={Microsoft Access Driver (*.mdb, *.accdb)};DBQ=C:\Recordings.accdb`

Error message is:
`Could not retrieve database metadata.  Please ensure that your driver, URL, username, and password are correct.`

I have created an ODBC driver in the 64-bit

**Comment by kirbykn2** [comment]
I had a similar error connecting to an Oracle db years ago.  I believe it
was a  user permissions issue.

On Mon, Dec 16, 2024 at 7:43 AM Frank Myhre ***@***.***>
wrote:

> Closed #6378
> <https://github.com/nextgenhealthcare/connect/discussions/6378> as
> outdated.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/nextgenhealthcare/connect/discussions/6378>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APRXWDYR5RD7XWH4CJB3K7T2F3DH3AVCNFSM6AAAAABTT2PW6OVHI2DSMVQWIX3LMV45UABFIRUXGY3VONZWS33OIV3GK3TUHI5E433UNFTGSY3BORUW63R3GE3DONBRGY4Q>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID:
> <nextgenhealthcare/connect/repo-discussions/6378/discussion_event/1674169@
> github.com>
>


-- 
Best,

Kirby Knight
| 231.735.4650 | ***@***.***
